components/dash-core-components/tests/integration/location/test_location_logout.py

```diff
@@ -33,15 +40,19 @@ def _insert_cookie(rep):
 
             return dcc.LogoutButton(id="logout-btn", logout_url="/_logout")
 
-    dash_dcc.start_server(app)
-    time.sleep(1)
-    dash_dcc.percy_snapshot("Core Logout button")
+    with pytest.warns(
```

Comment:
Need to add a condition `if add_initial_logout_button` for testing the deprecation notice otherwise it fails. Maybe better to just test the deprecation notice in a new test.

Reply:
Right. My worry is that `DeprecationWarning` is not being output when the `LogoutButton` is being used inside a callback—only when the `LogoutButton` is defined in `app.layout`.

This is the reason I was checking for the warning in the two separate scenarios. However, if we don't need to output a `DeprecationWarning` when the `LogoutButton` is used inside a callback, then I can just create a new test.

Reply:
@leeagustin can you please let us know if you added or are interested in adding the new test? thanks - @gvwilson